src/site/content/en/blog/ta-strategies/index.md

```diff
@@ -0,0 +1,266 @@
+
+As well as our role as developers, we all also use many other applications and devices. In this respect, we are the users and rely on all these systems to “just work”—that means we rely on countless developers to do their best to make their applications work. To turn this back around, as developers, we want to live up to this trust. So our first goal—whatever the project size—should always be to ship working software and thus serve our users. This means we must focus on our users' needs, especially in our tests. [Kent C. Dodds](https://kentcdodds.com/about) sums it up very well in [his post](https://kentcdodds.com/blog/static-vs-unit-vs-integration-vs-e2e-tests):
+
+<blockquote>
+  <p>
+    The more your tests resemble the way your software is used, the more confidence they can give you.
+  </p>
+  <cite>
+    by Kent C. Dodds
+  </cite>
+</blockquote>
+
+And I think this nails it. He describes it as becoming more confident in our tests. The closer we get to the users via choosing the corresponding testing type, the more confident we can trust our tests to have valid results. Or in other words, the higher up we climb the pyramid, the more confident we get. But wait, what is the pyramid? Don’t worry: we’ll cover this next.
+
+
+## Determining test strategies: how to choose a testing strategy
+
+Remember: as a first step, we want to determine which parts of our requirements need to be insured to be fulfilled, and how we want to achieve that. The goal behind that is to find out which types of testing to use, and then, with which granularity of testing we can reach the most confidence while also keeping an efficient cost structure. Many people approach this topic by using metaphors. Let’s take a deeper look at the most common ones, starting with the well-known classic.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/btUEUtD2bmqNwX2DcH4j.jpeg", alt="ALT_TEXT_HERE", width="800", height="450" %}
+
+
+### The classic: the test pyramid
+
+You will probably encounter the test automation pyramid as your initial metaphor as soon as you look up testing strategies. Mike Cohn introduced this concept in his book *Succeeding with Agile*. Later, the concept was expanded upon as the "[Practical Test Pyramid](https://martinfowler.com/articles/practical-test-pyramid.html)" by Martin Fowler. The visual representation of this pyramid is as follows:
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/iEtr9phjNjENxQlFfJYv.jpeg", alt="ALT_TEXT_HERE", width="800", height="450" %}
+
+As shown in this drawing, the test pyramid consists of three layers:
+
+1. **Unit**: You find these tests on the base layer of the pyramid because they are fast execution and simple to maintain. This is due to their isolation and targeting of the most minor units. See this as an example of a typical [unit test](https://github.com/leichteckig/phpmagazin-jest-example/blob/main/product.test.js) testing a very small product.
```

Comment:
```suggestion
1. **Unit**. You find these tests on the base layer of the pyramid because they are fast execution and simple to maintain. This is due to their isolation and targeting of the most minor units. See this as an example of a typical [unit test](https://github.com/leichteckig/phpmagazin-jest-example/blob/main/product.test.js) testing a very small product.
```